src/presentational/TeacherForm.jsx

```diff
@@ -87,85 +93,82 @@ class TeacherForm extends Component {
           <p>Before you get started setting up your first classroom, please answer the following questions about how you plan to use WildCam Lab in your teaching.</p>
         </div>
         <form onSubmit={this.handleSubmit}>
+
           <div className="form-group">
-           <label>Where do you teach?</label>
-           <input className="form-control"
-            type="text"
-            name="country"
-            placeholder="Insert a country"
-            autofocus="true"
-            value={this.state.country}
-            required
-            onChange={this.handleChange}/>
+            <label>Where do you teach?</label>
+            <Dropdown
+              required
+              name='country'
+              options={countries}
+              value='default'
+              onChange={this.handleChange}/>
           </div>
           <div className="form-group">
-           <label>In what educational setting do you plan to use this resource?</label>
-           <input className="form-control"
-            type="text"
-            name="setting"
-            placeholder="E.g. Classroom, home"
-            autofocus="true"
-            required
-            value={this.state.setting}
-            onChange={this.handleChange}/>
+            <label>In what educational setting do you plan to use this resource?</label>
```

Comment:
See above. `label` elements need to be explicitly associated with a single form control. See the examples at https://developer.mozilla.org/en/docs/Web/HTML/Element/label